### PR TITLE
Feature/service type sort/mhabermas

### PR DESCRIPTION
**Pull request by MichaelHabermas**
### Description

**Invited Collaborator(s) to Review
- Added person to review relating to feature/bug
- If not available, ask another reviewer on team

**What work was done?**
- fixed service tab service type sort

**Why was this work done?**
- enable user to organize data logs better

**What feature / user story is it for?**
(Link if possible) 
https://trello.com/c/UM3QaGpG/58-as-an-authorized-user-i-can-filter-the-service-logs-by-a-service-type-or-types

### Type of change
Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

Testing
- Has this feature been tested before conflict?
- Has this feature been tested after conflict?

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

File: src/components/common/ServicesTable/ServiceTable.js [hunk 1]
```diff
@@ -127,7 +128,7 @@ const ServicesTable = ({
           <Form.Item
             name="service_type_id"
             style={{ margin: 0 }}
-            initialValue={record.service_type_id}
```
Comment (ahmedseragcodes) [inline src/components/common/ServicesTable/ServiceTable.js:130]: recommending this stay as service_type_id

**Review by ahmedseragcodes — CHANGES_REQUESTED** [review]
Spoke with author, making one change to property for service_type_id to keep it that way instead of changing it

**Review by ErikBurdett — APPROVED** [review]
Looks good to me!

**Review by ahmedseragcodes — APPROVED** [review]
Agreed upon / previously discussed change, approved

**Pull request merged**